### PR TITLE
kernel: bump 6.1 to 6.1.68

### PR DESCRIPTION
**Pull request by graysky2**
Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.68

Removed upstreamed:
	generic/backport-6.1/795-v6.6-12-r8152-Rename-RTL8152_UNPLUG-to-RTL8152_INACCESSIBLE.patch[1]

Manually rebased:
	mediatek/patches-6.1/100-dts-update-mt7622-rfb1.patch

All other patches automatically rebased.

1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.1.68&id=3759e735562a31e44fee825498f05c06e64b25a8

Build system: x86/64
Build-tested: x86/64/AMD Cezanne
Run-tested: x86/64/AMD Cezanne

### CODE REVIEW

**Comment by graysky2** [comment]
@RyderLee1110 - can you please confirm my manual rebase to your patch is correct?

**Comment by pkgadd** [comment]
Build-tested (on x86_64):
- ath79
- ipq40xx
- ipq806x
- ipq807x
- x86_64

Runtime-tested:
- ipq807x (xiaomi ax3600)
- x86_64 (j1900)
  ```
  Host/Kernel/OS "gateprotect" running Linux 6.1.68 x86_64 [ OpenWrt SNAPSHOT r24660-c22aa0be3e ]
  System         R & S Cybersecurity gateprotect GmbH GP-7543
  CPU Info       4x Intel Celeron J1900 @ 1024 KB cache flags( sse3 ht nx lm vmx ) clocked at [ 2004.807 MHz ]
  Videocard      Intel Atom Processor Z36xxx/Z37xxx Series Graphics & Display  tty resolution (  )
  Network cards  4x Intel I211 Gigabit Network Connection, at ports: d000 c000 b000 a000 
  Processes 121 | Uptime 41min | Memory 89.4/3832.1MB | HDD 32GB SATA Flash Size 32GB (0%used) | Infobash v3.60
  ```

```
Tested-by: Stefan Lippers-Hollmann <s.l-h@gmx.de> #x86_64 (j1900), ipq807x (ax3600)
```

Works fine for me :)

File: target/linux/mediatek/patches-6.1/100-dts-update-mt7622-rfb1.patch [hunk 1]
```diff
@@ -35,8 +35,9 @@
  		};
  	};
  
- 	memory {
+-	memory@40000000 {
 -		reg = <0 0x40000000 0 0x20000000>;
++	memory {
```
Comment (DragonBluep) [inline target/linux/mediatek/patches-6.1/100-dts-update-mt7622-rfb1.patch:40]: Just keep `memory@40000000`. ref:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit?h=v6.1.68&id=c2adea319187cb6d7e27cacd9aba24f980b7c364

Comment (graysky2) [reply target/linux/mediatek/patches-6.1/100-dts-update-mt7622-rfb1.patch]: Please review my new force pushed commit.

Comment (DragonBluep) [reply target/linux/mediatek/patches-6.1/100-dts-update-mt7622-rfb1.patch]: Hi! We need to keep the RAM size change. So we just need to edit the original patch to change `memory` to `memory@40000000`.

From
```
 	};

 	memory {
-		reg = <0 0x40000000 0 0x20000000>;
+		reg = <0 0x40000000 0 0x40000000>;
 	};

 	reg_1p8v: regulator-1p8v {
```
to
```
 	};

 	memory@40000000 {
-		reg = <0 0x40000000 0 0x20000000>;
+		reg = <0 0x40000000 0 0x40000000>;
 	};

 	reg_1p8v: regulator-1p8v {
```

Comment (graysky2) [reply target/linux/mediatek/patches-6.1/100-dts-update-mt7622-rfb1.patch]: I see now... please review my new force pushed commit.

**Review by DragonBluep — APPROVED** [review]

**Comment by PolynomialDivision** [comment]
Looks good to me. I would merge except you still want feedback from @ryderlee1110?

**Comment by graysky2** [comment]
@PolynomialDivision - based on @DragonBluep's feedback, I think the PR is good as-is.  What do you think?

**Comment by PolynomialDivision** [comment]
I'm fine with it. Thanks!

**Comment by PolynomialDivision** [comment]
Thanks! Rebased on top of main and merged!

**Pull request merged**